.github/workflows/test.yml

```diff
@@ -161,7 +161,7 @@ env:
   FF_TFM_DEAL_CANCELLATION_ENABLED: ${{ vars.FF_TFM_DEAL_CANCELLATION_ENABLED }}
   FF_PORTAL_FACILITY_AMENDMENTS_ENABLED: ${{ vars.FF_PORTAL_FACILITY_AMENDMENTS_ENABLED }}
   FF_TFM_SSO_ENABLED: ${{ vars.FF_TFM_SSO_ENABLED }}
-  GEF_DEAL_VERSION: ${{ vars.GEF_DEAL_VERSION }}
+  GEF_DEAL_VERSION: 1
```

Comment:
please can this be reverted.  We should change the var for GEF_DEAL_VERSION  instead of hard coding 1 here